tests/runMPITest.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import subprocess
+
+
+def runMPITest(nbProcesses, fileAbspath):
+    subprocessError = None
+    try:
+        result = subprocess.run(args=['mpirun', '-n', str(nbProcesses), 'python', fileAbspath], stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True, universal_newlines=True)
+        print(result.stdout)
+    except subprocess.CalledProcessError as error:
+        print(error.stdout)
+        subprocessError = AssertionError(error.stderr)
+    if subprocessError:
+        raise subprocessError
```

Comment:
Le if me semble superflu

```suggestion
    except subprocess.CalledProcessError as error:
        print(error.stdout)
        subprocessError = AssertionError(error.stderr)
        raise subprocessError
```

Reply:
Ou même juste "raise" je crois que ça marche pour renvoyer l'erreur attrapée. 

J'ai fait comme ça pour rendre le message d'erreur plus lisible. Sinon il te raconte qu'il a levé un CalledProcessError, puis qu'une autre erreur a été levée pendant qu'il gérait la première etc. Comme ça le CalledProcessError disparait complètement du message d'erreur, ça me semble plus clair. Qu'est-ce que tu en penses ?

Reply:
Je comprends pas ce que la modification que je propose change dans le code

Reply:
Tiens regarde l'exemple ci-dessous. Relancer l'erreur en dehors du except permet de faire disparaître la première partie du message d'erreur. C'est tout ^^. Ca évite de faire apparaître un CalledProcessError dans le message d'erreur.

try:
...  raise AssertionError("toto")
except:
...  raise NotImplementedError("tata")

Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
AssertionError: toto

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
NotImplementedError: tata

Reply:
ah c'est marrant ca. j'ignorais!